janusgraph-server/src/test/resources/janusgraph-server-no-auto.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 JanusGraph Authors
```

Comment:
(nitpick) I think `-no-auto` and `-cfg` aren't descriptive enough to explain the purpose of these config files. Could you make the names a bit more descriptive?
Isn't the important part of this config file for example that it has no `graphManager` configured?